draft-jhoyla-req-mtls-flag.md

```diff
@@ -144,6 +144,49 @@ terminate the connection with a fatal `illegal_parameter` alert.
 
 # Security Considerations
 
+## Authentication Properties
+
+TLS client authentication is bound to a particular TLS session. This means that
+a malicious server cannot manipulate a client in such a way that they can
+impersonate said client to any other server. This is due to the matching
+sessions property of TLS.
```

Comment:
let's add an informative reference for "matching sessions property of TLS"? https://eprint.iacr.org/2020/1044 comes to mind, but pick your favorite :)